wbooks/src/app/screens/Settings/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import styles from './styles';
+
+const SettingsScreen = () => (
```

Comment:
Change this arrow function for a function.

Reply:
done